code/modules/flufftext/Hallucination.dm

```diff
@@ -898,8 +1036,18 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 		if("ratvar")
 			target.playsound_local(target, 'sound/machines/clockcult/ark_deathrattle.ogg', 50, FALSE, pressure_affected = FALSE)
 			target.playsound_local(target, 'sound/effects/clockcult_gateway_disrupted.ogg', 50, FALSE, pressure_affected = FALSE)
-			sleep(27)
-			target.playsound_local(target, 'sound/effects/explosion_distant.ogg', 50, FALSE, pressure_affected = FALSE)
+			addtimer(CALLBACK(
+				target,
+				/mob/.proc/playsound_local,
+				target,
+				'sound/effects/explosion_distant.ogg',
+				50,
+				FALSE,
+				/* frequency = */ null,
+				/* falloff_exponential = */ null,
+				/* channel = */ null,
+				/* pressure_affected = */ FALSE
```

Comment:
?????

Reply:
CALLBACK is a macro for `new /datum/callback`. Thus, it does not support proper named arguments, and needs the hack I use here in order to make it obvious what the variables actually do.

Reply:
But you have callbacks with the same proc just a bit above without the additional vars

Reply:
Ah excuse me I didn't see pressure_effected